xbmc/pvr/PVRManager.h

```diff
@@ -36,23 +35,42 @@ namespace PVR
 
   enum class PVREvent
   {
+    None = 0,
+
     // PVR Manager states
-    ManagerError = 0,
+    ManagerError,
```

Comment:
This used to zero, there is nothing dependent on the old value all being +1 now? DB entries etc? Logically there shouldn't be.

Reply:
No, no further deps.

Reply:
Cool

Reply:
manager state will never be persisted.

Reply:
Wait, I can even revert that change as the newly introduced "None" is only a leftover from an earlier iteration of the commit.

Reply:
done